src/transformer_util.ts

```diff
@@ -301,11 +302,9 @@ function resetNodeTextRangeToPreventDuplicateComments<T extends ts.Node>(node: T
   if (node.kind === ts.SyntaxKind.PropertyDeclaration) {
     allowTextRange = false;
     const pd = node as ts.Node as ts.PropertyDeclaration;
-    // TODO(tbosch): Using pd.initializer! as the typescript typings before 2.4.0
-    // are incorrect. Remove this once we upgrade to TypeScript 2.4.0.
     node = ts.updateProperty(
-               pd, pd.decorators, pd.modifiers, resetTextRange(pd.name) as ts.PropertyName, pd.type,
-               pd.initializer!) as ts.Node as T;
+               pd, pd.decorators, pd.modifiers, resetTextRange(pd.name) as ts.PropertyName,
+               pd.questionToken, pd.type, pd.initializer) as ts.Node as T;
```

Comment:
does the 2.4 updateProperty accept the questionToken here?

Reply:
This change is not meant to be backwards compatible with 2.4. I am confused why are you asking about 2.4?

Reply:
As discussed, this change does not have backwards compatibility with TS2.4. I will update the 2.4 branch to the current head before submitting this. Chuck will work on making tsickle 2.4 + 2.5 compatible separately.